saleor/graphql/attribute/types.py

```diff
@@ -384,8 +384,11 @@ class Meta:
 
 class AttributeInput(BaseInputObjectType):
     slug = graphene.String(required=True, description=AttributeDescriptions.SLUG)
+    value_names = NonNullList(
+        graphene.String, required=False, description="Names of the values."
+    )
     values = NonNullList(
-        graphene.String, required=False, description=AttributeValueDescriptions.SLUG
+        graphene.String, required=False, description="Slugs of the values."
```

Comment:
Maybe worth to provide more info here?  Like:
"A slug identifying the attributeValues associated with the Attribute. When specified, it filters the results to include only records with one of the given values."
 and something similar for `Names` part?